src/components/camera-tool.js

```diff
@@ -39,8 +39,8 @@ const CAPTURE_WIDTH = isMobileVR ? 640 : 1280;
 const CAPTURE_HEIGHT = isMobileVR ? 360 : 720;
 const RENDER_WIDTH = 1280;
 const RENDER_HEIGHT = 720;
-const CAPTURE_DURATIONS = allowVideo ? [0, 3, 7, 15, 30, 60, Infinity] : [0];
-const DEFAULT_CAPTURE_DURATION = allowVideo ? 3 : 0;
+const CAPTURE_DURATIONS = [3, 7, 15, 30, 60, Infinity];
+const DEFAULT_CAPTURE_DURATION = 3;
 const COUNTDOWN_DURATION = 3;
```

Comment:
I suggest losing the countdown especially for photos. The delay between when I click to when it starts feels frustrating/unnecessary to me and it prevents quick/candid capture of dynamic events (snapping while a video is on a particular frame, snapping while something is moving, snapping while people have configured themselves in a particular way). Might be useful as a user pref. Otherwise I suggest we gather user feedback to get evidence of one or the other if we disagree.

Reply:
the reason i added the countdown is because there is no way to pose for a picture, since you need to be using your hand to click the button. right now, the way to take a point in time photo is to point-and-shoot. i guess i could add an option for the timer in the menu tho, like the audio toggle.

Reply:
Ah, that makes sense. Would be nice to have both options, but not worth blocking this PR if it's difficult